### ISSUE TITLE
C_TooltipInfo.GetHyperlink with unit:Creature gives incomplete/incorrect data

### ISSUE THREAD

**Issue by shooter-z** [Stanzilla/WoWUIBugs#297]
If `unit:Creature-0-0-0-0-id` link is passed to C_TooltipInfo.GetHyperlink results given are not complete, and have incorrect data.
For npc 3310 Instead of 
```
Doras
Wind Rider Master
Level 70 (Elite)
Orgrimmar
PvP
```
only
```
Wind Rider Master
Level ?? Humanoid (Elite)
```
is returned. Npc data is already known by client, since mouse over tooltip for the npc is showing correct info. Running the code at the same time gives incorrect data shown above.

Example: `/dump C_TooltipInfo.GetHyperlink("unit:Creature-0-0-0-0-3310")`
Tested on 10.0.2.45969

**Comment by Nevcairiel** [comment]
Returning this information is not guaranteed. The tooltip is designed to show information about units around you, and if the information is not cached or otherwise available there is no guaranteed that it'll be available.

While this may have worked in the past, its not exactly part of the API contract to retrieve generic information about fake units that don't actually exist. So you may have to find another means to this problem.

**Comment by Meorawr** [comment]
Enquired about this specific issue - it's not an intentional breakage.

**Comment by Meorawr** [comment]
Fixed in 10.0.2.46144.
